draft-ietf-quic-transport.md

```diff
@@ -2089,93 +1826,358 @@ Connection ID:
+|  [Gap N (8)]  |       [ACK Block N Length (8/16/32/64)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #ack-block-format title="ACK Block Section"}
+
+The fields in the ACK Block Section are:
+
+First ACK Block Length:
+
+: An unsigned packet number delta that indicates the number of contiguous
+  additional packets being acknowledged starting at the Largest Acknowledged.
+
+Gap To Next Block (opt, repeated):
+
+: An unsigned number specifying the number of contiguous missing packets from
+  the end of the previous ACK block to the start of the next.  Repeated "Num
+  Blocks" times.
+
+ACK Block Length (opt, repeated):
+
+: An unsigned packet number delta that indicates the number of contiguous
+  packets being acknowledged starting after the end of the previous gap.
+  Repeated "Num Blocks" times.
+
+
+### Timestamp Section {#timestamp-section}
+
+The Timestamp Section contains between zero and 255 measurements of packet
+receive times relative to the beginning of the connection.
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+
+| [Delta LA (8)]|
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                    [First Timestamp (32)]                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA 1(8)]| [Time Since Previous 1 (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA 2(8)]| [Time Since Previous 2 (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+                       ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Delta LA N(8)]| [Time Since Previous N (16)]  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #timestamp-format title="Timestamp Section"}
+
+The fields in the Timestamp Section are:
+
+Delta Largest Acknowledged (opt):
+
+: An optional 8-bit unsigned packet number delta specifying the delta between
+  the largest acknowledged and the first packet whose timestamp is being
+  reported.  In other words, this first packet number may be computed as
+  (Largest Acknowledged - Delta Largest Acknowledged.)
+
+First Timestamp (opt):
+
+: An optional 32-bit unsigned value specifying the time delta in microseconds,
+  from the beginning of the connection to the arrival of the packet indicated by
+  Delta Largest Acknowledged.
+
+Delta Largest Acked 1..N (opt, repeated):
+
+: This field has the same semantics and format as "Delta Largest Acknowledged".
+  Repeated "Num Timestamps - 1" times.
+
+Time Since Previous Timestamp 1..N(opt, repeated):
+
+: An optional 16-bit unsigned value specifying time delta from the previous
+  reported timestamp.  It is encoded in the same format as the ACK Delay.
+  Repeated "Num Timestamps - 1" times.
+
+The timestamp section lists packet receipt timestamps ordered by timestamp.
+
+
+## STREAM Frame {#frame-stream}
```

Comment:
You moved this into the middle of the ACK frame description (see the time format section below).  I'll fix this.